kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/KafkaStreamsApp.java

```diff
@@ -294,4 +295,12 @@ private void preCreateTopics(Map<String, Object> properties) {
           (String) properties.get(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG), topics);
     }
   }
+
+  private void evaluateShutdownDuration() {
```

Comment:
Can we change it to `getShutdownDuration` and then assign it on L169 to `this.shutdownDuration`?